packages/version-select/util.test.ts

```diff
@@ -89,3 +93,29 @@ describe('removeVersionFromPath', () => {
     )
   })
 })
+
+describe('getTargetPath', () => {
```

Comment:
Should we add a test case here with a non-nested base path?

Reply:
added!